services/sonar/sonar-base.js

```diff
@@ -70,8 +70,11 @@ module.exports = class SonarBase extends BaseJsonService {
     } else {
       schema = modernSchema
       url = `${server}/api/measures/component`
+      // componentKey query param was renamed in version 6.6
+      const componentKey =
+        parseFloat(sonarVersion) >= 6.6 ? 'component' : 'componentKey'
       qs = {
-        componentKey: component,
+        [componentKey]: component,
```

Comment:
Long term it'd probably be better to refactor this out and repurpose `isLegacyVersion` to have a single helper that provides the url, schema, and query string. However, this will work for now so happy to proceed to address the immediate blocking issue